stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -63,7 +56,7 @@ add_swift_target_library(swiftTensorFlow ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_
   SWIFT_MODULE_DEPENDS_FREEBSD Glibc
   SWIFT_MODULE_DEPENDS_CYGWIN Glibc
   SWIFT_MODULE_DEPENDS_HAIKU Glibc
-  ${TENSORFLOW_DEPENDS_PYTHON}
+  SWIFT_MODULE_DEPENDS Python
```

Comment:
For now, making `Python` an explicit (rather than informal) dependency of `TensorFlow` is good.

However, only a small portion of `TensorFlow` depends on `Python`. Eventually, we may want to add a `TensorFlow+Python` module. [Related discussion on "cross-import overlays"](https://forums.swift.org/t/cross-import-overlays/30868).

Perhaps the `Python` dependency issue will go away or change in nature when `TensorFlow` is no longer built as part of the standard library: https://github.com/tensorflow/swift-apis/issues/488.

Reply:
It definitely would be nice to have the ability to not have `Python` used.  However, I think that we currently do not have that ability so this is simply structuring the code to be simpler in light of that.  I think that when the time comes, providing a user controllable knob to enable the dependency is a better approach.  The user knows what they want, we do not need to infer it, and that still would avoid the ad-hoc state tracking.

Reply:
Yep, I'm totally in favor of an explicit `Python` dependency! Just wanted to share some future direction ideas.